### ISSUE TITLE
[Accessibility] No role for control defined: Button

### ISSUE THREAD

**Issue by amberhinds** [woocommerce/woocommerce-gateway-amazon-pay#334]
## Audit Information
This issue came from an audit WooCommerce had Equalize Digital complete in December 2025. I'm moving the issue out of Linear (WooA11Y-415) to this repo.

- EQD Audit ID: WooExt-510
- Component/Page: Single Products
- Affected Element: Amazon Pay button
- URL where issue was discovered: https://apay-woo.mystagingwebsite.com/product/lightweight-wool-table/

## WCAG Compliance

- Success Criterion: 4.1.2 Name, Role, Value
- WCAG Version: 2
- Level: A
- Severity: Medium

## Issue Description

The element is missing a semantic role that programmatically identifies it as a button, so its function as an interactive control is not clearly conveyed to assistive technologies.

### Impacted Populations

Screen reader users (blind/low vision and neurodivergent people)

## Recommended Fix

Instead of using a `<div>`, use the semantic HTML `<button>` tag for this element to make it match its function.

If you cannot replace a div with a button, it is possible to remediate a div to function as a button with ARIA. The div would need to be given a button role and `tabindex="0"` to ensure that it can be focused with the keyboard. You would also need to write JavaScript to ensure that the div will function properly as a button for keyboard users.

For more information on turning a div into a button, refer to: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Use html semantic ARIA role to indicate the role: button

## Code Example
```
<div class="amazonpay-button-container amazonpay-button-enabled amazonpay-button-container-rows" id="maxo-button-wrapper-1764832198951"><div class="amazonpay-button-view1 amazonpay-button-view1-gold" tabindex="0"><picture class="amazonpay-button-logo"><img src="https://m.media-amazon.com/images/G/01/AmazonPay/Maxo/logo._CB452516594_.svg" alt="Amazon Pay"></picture><picture class="amazonpay-button-sandbox-logo"><img src="https://m.media-amazon.com/images/G/01/AmazonPay/Maxo/sandbox_icon._CB452516595_.svg" alt=""></picture><picture class="amazonpay-button-chevrons"><img src="https://m.media-amazon.com/images/G/01/AmazonPay/Maxo/AmazonPay_button_chevron._CB1558391205_.svg" alt=""></picture></div><div class="amazonpay-button-view2"></div><div class="amazonpay-button-view3"><picture class="amazonpay-button-microtext"><img src="https://static-na.payments-amazon.com/assets/maxo/microtext/en_US.svg" aria-label="Use your Amazon Pay Sandbox test account"></picture></div></div>
```
## Screenshots

https://drive.google.com/file/d/1O_OsZoPX498_ir8HdKdrGdSL5M46R4eU/view?usp=drivesdk
